homeassistant/components/cert_expiry/sensor.py

```diff
@@ -36,20 +31,21 @@
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
```

Comment:
Rename this function and just `pass` inside.
```suggestion
async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
```